packaging/vagrant/Vagrantfile

```diff
@@ -171,7 +171,32 @@ EOF2
     (cd shadow-4.6 && ./configure && sudo make install)
 
     # Install Buildah.
-    sudo yum -y install buildah
+    sudo yum -y install \
```

Comment:
I like to arrange these in alphabetical order and lined up after `install`, e.g.:

```
sudo yum -y install bar \
                    baz \
                    foo
```